tools/sdk-ai-bots/azure-sdk-qa-bot-evaluation/run.py

```diff
@@ -430,6 +443,10 @@ def establish_baseline(args: argparse.Namespace, all_results: dict[str, Any]) ->
                 }
         
         output_file_dir = asyncio.run(prepare_dataset(args.test_folder, args.prefix, args.is_bot))
+        if output_file_dir is None:
+            print(f"No test data file to process. exit")
```

Comment:
Better to use [logging](https://docs.python.org/3/library/logging.html) to add log info.